### DISCUSSION TITLE
can we change nl_abs_tol over time?

### DISCUSSION THREAD

**Discussion by mortezaaesmaeilpour** [idaholab/moose#23318] [Q&A General] [answered]
Hi

Is it possible to define absolute tolerance (nl_abs_tol) as a time function? (vary over time)

**Comment by GiudGiud** [comment, marked as answer]
Hello

We currently cannot. It's something we have had a lot of interest for, but there's no dedicated funding/resource to do that right now

Guillaume

**Reply by GiudGiud** [reply]
If you use automatic scaling and set compute_scaling_once = false, the absolute nonlienar tolerance will vary along with the problem , so that is some way to get it to vary if it s becoming too loose with time.

The other option is to rely on relative tolerances as much as possible
